client/src/containers/auth-layout/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { FC, PropsWithChildren } from "react";
+
+import { cn } from "@/lib/utils";
+
+type AuthLayout = PropsWithChildren<{
+  className?: HTMLDivElement["className"];
+}>;
+
+const AuthLayout: FC<AuthLayout> = ({ className, children }) => {
+  return (
+    <div className="container px-10">
```

Comment:
I have my doubts about this padding, the left side doesn't need that much space.